src/Microsoft.AspNetCore.Sockets.Abstractions/ConnectionContext.cs

```diff
@@ -22,6 +22,6 @@ public abstract class ConnectionContext
         public abstract ConnectionMetadata Metadata { get; }
 
         // TEMPORARY
-        public abstract IChannelConnection<Message> Transport { get; set; }
+        public abstract IChannelConnection<byte[]> Transport { get; set; }
```

Comment:
So with this, `Message` is dead yes? Why keep the `T` in the text protocol then? We can just make it `{length}:{payload}` now rather than leaving a vestigial "type" flag in.